<?xml version="1.0"?>
<launch>
    <!-- Parameters defining the discretized world width and height -->
    <param name="/world_width"     value="12" />
    <param name="/world_height"    value="16" />

    <!-- Parameters to define the sleeping position -->
    <param name="/sleep_x_coord" value="-5" />
    <param name="/sleep_y_coord" value="5" />

    <!-- Parameter to define the robot maximul level of efforts before going to sleep -->
    <param name="/fatigue_threshold" value="6" />

    <!-- Parameter to define the robot will wait in the Track Ball state when stopping detecting the ball -->
    <param name="/maximum_dead_time" value="5" />

    <!-- Parameter to define the maximum time for which the robot will remain in the Play behavior -->
    <param name="/max_play_time" value="600" />

    <!-- Parameter to define the maximum time for which the robot will remain in the Explore state -->
    <param name="/max_explore_time" value="180" />

    <!-- Parameter to define rate of checking the target reachability -->
    <param name="/target_checking_rate" value="40" />

    <!-- Parameter to define the robot maximul linear velocity (safety reason) -->
    <param name="/max_speed" value="0.5" />


    <!-- Include the launch file for the Gazebo setup -->
    <include file="$(find robot_simulation_state_machines)/launch/sim_w1.launch" >
    </include>


    <include file="$(find robot_simulation_state_machines)/launch/gmapping_setting.launch">
    </include>

    <include file="$(find robot_simulation_state_machines)/launch/move_base_setting.launch">
    </include>


    <include file="$(find explore_lite)/launch/explore.launch">
    </include>


</launch>
